### DISCUSSION TITLE
Problem with panw module - filebeat

### DISCUSSION THREAD

**Discussion by mariusz326** [Security-Onion-Solutions/securityonion#8847] [Unsupported Versions]
Hello,

I have a problem with the panw module in filebeat. Below I enclose the module configuration.

In /opt/so/saltstack/local/pillar/minions/so3_standalone.sls file I added below:

` third_party_filebeat:
    modules:
      panw:
        panos:
          enabled: true
          var.input: udp
          var.syslog_host: 0.0.0.0
          var.syslog_port: 9002`

In /opt/so/saltstack/local/pillar/global.sls I added below:

` filebeat:
   third_party_filebeat:
     modules:
       panw:
         panos:
           enabled: true`

docker ps command return below:
`a9834b28707c   so3:5000/security-onion-solutions/so-filebeat:2.3.160             "/usr/local/bin/dock…"   12 seconds ago   Up 10 seconds           0.0.0.0:514->514/tcp, 0.0.0.0:514->514/udp, 0.0.0.0:5066->5066/tcp, 0.0.0.0:9002->9002/udp`

After configuration I execute below commands:

`sudo so-filebeat-restart
sudo salt-call state.apply common`

Filebeat container gets ERROR status after some time. In filebeat logs we can below error:

`{"log.level":"error","@timestamp":"2022-10-04T07:29:27.117Z","log.origin":{"file.name":"instance/beat.go","file.line":1051},"message":"Exiting: Failed to start crawler: creating module reloader failed: error checking input configuration: No paths were defined for input accessing config","service.name":"filebeat","ecs.version":"1.6.0"}`

What went wrong during setup?

**Comment by Sahale** [comment]
Hey!

If I'm not mistake, you should either edit `global.sls` to enable panw module on all nodes, or edit pillar file (`so3_standalone.sls` in your case) of certain nodes to enable module only on them.
In your's `global.sls` there is no `var.syslog_port` variable.
I'd started with deleting pans settings in `global.sls` file and then check if it works correctly. Or add `var.syslog_port` to `global.sls`.

**Reply by mariusz326** [reply]
After deleting the configuration from global.sls I have the same error as before. I also tried adding the configuration to global.sls with the var.syslog_port variable:

`filebeat:
  third_party_filebeat:
    modules:
      panw:
        panos:
          enabled: true
          var.syslog_port: 9002`

When I try to restart, filebeat (so-filebeat-restart) gets an error:
`Comment: Created container 'so-filebeat'. Failed to start container 'so-filebeat': 'Error 500: driver failed programming external connectivity on endpoint so-filebeat`

**Reply by weslambert** [reply]
Are you able to find any clues by running `docker logs so-filebeat`?
